sdk/storage/Azure.Storage.Files.DataLake/src/Models/ChangeAccessControlListResultFailedEntry.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+namespace Azure.Storage.Files.DataLake.Models
+{
+    /// <summary>
+    /// Represents an entry that failed to update Access Control List.
+    /// </summary>
+    public class ChangeAccessControlListResultFailedEntry
```

Comment:
Same comment about One SDK probably wanting to rename this, and adding a static method in ```models/DataLakeModelFactory.cs```